libs/cimit-service/src/test/java/uk/gov/di/ipv/core/library/pact/ContractTest.java

```diff
@@ -0,0 +1,1364 @@
+                    }
+                  ],
+                  "address": [
+                    {
+                      "uprn": null,
+                      "organisationName": null,
+                      "subBuildingName": null,
+                      "buildingNumber": null,
+                      "buildingName": null,
+                      "dependentStreetName": null,
+                      "streetName": null,
+                      "doubleDependentAddressLocality": null,
+                      "dependentAddressLocality": null,
+                      "addressLocality": null,
+                      "postalCode": "CH62 6AQ",
+                      "addressCountry": null
+                    }
+                  ],
+                  "drivingPermit": [
+                    {
+                      "personalNumber": "DOEDO861281JF9DH",
+                      "issueNumber": null,
+                      "issuedBy": "DVLA",
+                      "issueDate": null,
+                      "expiryDate": "2028-08-07",
+                      "fullAddress": "102 TEST ROAD,WIRRAL,CH62 6AQ"
+                    }
+                  ]
+                },
+                "evidence": [
+                  {
+                    "type": "IdentityCheck",
+                    "txn": "bcd2346",
+                    "strengthScore": 3,
+                    "validityScore": 0,
+                    "ci": [
+                      "TEST-CI-CODE"
+                    ],
+                    "activityHistoryScore": 0,
+                    "failedCheckDetails": [
+                      {
+                        "checkMethod": "vri",
+                        "identityCheckPolicy": "published",
+                        "activityFrom": null
+                      },
+                      {
+                        "checkMethod": "bvr",
+                        "biometricVerificationProcessLevel": 3
+                      }
+                    ]
+                  }
+                ]
+              },
+              "exp": 4070909400
+            }
+            """;
+
+    // If we generate the signature in code it will be different each time, so we need to generate a
+    // valid signature (using https://jwt.io works well) and record it here so the PACT file doesn't
+    // change each time we run the tests.
+    private static final String DVLA_VC_WITH_CI_AND_INVALID_ISSUER_SIGNATURE =
+            "8c6iIjlh7JipvbhkOBesiumnFp_EcZpAkBuOpVxNIdAO68ZUmxMYW4MpVi7jz0-dbMGOgKHROIPQZ4yw-wsa_Q"; // pragma: allowlist secret
+    private static final String DVLA_VC_WITH_CI_AND_INVALID_ISSUER_JWT =
+            new PactJwtBuilder(
+                            VALID_VC_HEADER,
+                            DVLA_VC_WITH_CI_AND_INVALID_ISSUER,
+                            DVLA_VC_WITH_CI_AND_INVALID_ISSUER_SIGNATURE)
+                    .buildJwt();
+
+    // 2099-01-01 00:00:00 is 4070908800 in epoch seconds
+    // From DCMAW-5477-AC1
```

Comment:
What is this?

Reply:
It's a reference for where this test data comes from (it's also used in the [dcmaw contract tests](https://github.com/govuk-one-login/ipv-core-back/blob/2993d0a4c37b00b3a77fe62e945b4abd8ee5df24/lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/dcmawCri/ContractTest.java#L2037))